media/redesign/stylus/components/wiki/revision-list.styl

```diff
@@ -97,3 +97,7 @@ html[dir=rtl] .revision-list-controls:before {
     width: 460px;
 }
 
+/* make the Compare selected revisions button align to layout */
+.revision-list-controls .fixed {
+    left: 24px;
```

Comment:
I realized that I can just edit the DOM in Inspector and set dir ="rtl" - but don't let that take away from you enjoying testing on your local system heh.